config/repos.yml

```diff
@@ -1,6 +1,7 @@
 master:
   amazon_ssa_support:
   container-amazon-smartstate:
+    has_real_releases: true
```

Comment:
This was left out on purpose in #116 because we do want to create branches.

Reply:
Oh interesting...let's instead comment it in the repos.yml file with an explanation, otherwise it's confusing.